data/human/human missions.txt

```diff
@@ -11281,3 +11281,164 @@ mission "Patch Stone Waiting"
+				decline
+
+
+mission "Pact Supplies Convoy"
+	job
+	name "Transport weapons to <planet>"
+	description "The local militia is requesting merchant captains to participate in a convoy supplying <cargo> to <destination>. Available captains are to report to the quartermaster's office. Payment is <payment>."
+	cargo "weapons" 20
+	destination "Longjump"
+	source
+		attributes "dirt belt"
+		attributes "factory"
+		not planet "New Sahara"
+	to offer
+		or
+			has "ships: Transport"
+			has "ships: Light Freighter"
+			has "ships: Heavy Freighter"
+			has "ships: Utility"
+		not "Pact Supplies Escort: completed"
+	on accept
+		conversation "Pact Supplies"
+	npc accompany save
+		government "Merchant"
+		personality escort timid
+		fleet "Pact Supplies"
+	npc
+		government "Pirate"
+		personality plunders waiting
+		fleet "Small Southern Pirates" 2
+	on visit
+		dialog `You have reached <planet>, but you're missing something! Either you left part of the convoy behind, or not all of the cargo is in the system.`
+	on complete
+		"assisted the free worlds" ++
+		payment 35000 50
+		dialog `After you land at <planet>'s spaceport, a militia officer comes to inspect your ship before giving it the all-clear to be unloaded. "Thank you for your service. I will make sure that the <Pact / Free Worlds> remembers your assistance." He hands you your payment of <payment> and makes his way to the next convoy ship.`
+
+
+mission "Pact Supplies Escort"
+	job
+	name "Escort weapons convoy to <planet>"
+	description "The local militia is requesting merchant captains to escort a convoy supplying weapons to <destination>. Available captains are to report to the quartermaster's office. Payment is <payment>."
+	destination "Longjump"
+	source
+		attributes "dirt belt"
+		attributes "factory"
+		not planet "New Sahara"
+	to offer
+		or
+			has "ships: Interceptor"
+			has "ships: Light Warship"
+			has "ships: Medium Warship"
+			has "ships: Heavy Warship"
+			has "ships: Superheavy"
+		not "Pact Supplies Convoy: completed"
+	on accept
+		conversation "Pact Supplies"
+
+	npc accompany save
+		government "Merchant"
+		personality escort timid
+		fleet "Pact Supplies"
+	npc
+		government "Pirate"
+		personality plunders waiting
+		fleet "Small Southern Pirates" 2
+	on visit
+		dialog `You have reached <planet>, but you left part of the convoy behind! Better depart and wait for them to arrive in this star system.`
+	on complete
+		"assisted the free worlds" ++
+		payment 35000 50
+		dialog `After you land at <planet>'s spaceport, a militia officer comes to greet you. "Thank you for your service. I will make sure that the <Pact / Free Worlds> remembers your assistance." He hands you your payment of <payment>, and makes his way to the next convoy ship.`
```

Comment:
I notice that all your generic militia officers are male. Maybe mix it up a bit or use they/their as pronouns instead?

Reply:
No, the militia officer who informs you of the job offer is female.

Reply:
Ah, I obviously can't read.